error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "pawanbiker21@gmail.com" (type string) at path "_id" for model "User"
    at model.Query.exec (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4498:21)
    at model.Query.Query.then (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/pawanbiker21@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-06-16T00:51:02.311Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "_emailId=pawanbiker21@gmail.com" (type string) at path "_id" for model "User"
    at model.Query.exec (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4498:21)
    at model.Query.Query.then (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/_emailId=pawanbiker21@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-06-16T00:51:44.440Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "search" (type string) at path "_id" for model "User"
    at model.Query.exec (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4498:21)
    at model.Query.Query.then (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/search/?emailId=pawanbiker21@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-06-16T01:02:58.795Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "search" (type string) at path "_id" for model "User"
    at model.Query.exec (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4498:21)
    at model.Query.Query.then (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/search/?emailId=pawanbiker21@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-06-16T01:04:54.792Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "search" (type string) at path "_id" for model "User"
    at model.Query.exec (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4498:21)
    at model.Query.Query.then (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/search/?emailId=pawanbiker21@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-06-16T01:12:07.441Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "search" (type string) at path "_id" for model "User"
    at model.Query.exec (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4498:21)
    at model.Query.Query.then (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/search/?emailId=pawanbiker21@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-06-16T01:19:17.599Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "pawanbiker21@gmail.com" (type string) at path "_id" for model "User"
    at model.Query.exec (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4498:21)
    at model.Query.Query.then (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/pawanbiker21@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-06-16T01:19:39.614Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "pawanbiker21@gmail.com" (type string) at path "_id" for model "User"
    at model.Query.exec (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4498:21)
    at model.Query.Query.then (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/pawanbiker21@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-06-16T01:19:58.735Z"}
error: Could not connect to the database. connect ECONNREFUSED 127.0.0.1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:27017\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:847:32)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12)\n    at database (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:4:15)\n    at Object.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1255:14)\n    at Module._compile (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/pirates/lib/index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1309:10)\n    at Object.newLoader [as .js] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/pirates/lib/index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:1113:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:960:12)\n    at Function.runMain (node:internal/modules/run_main:83:12)\n    at Object.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/node/src/_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1255:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1309:10)\n    at Module.load (node:internal/modules/cjs/loader:1113:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:960:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)\n    at node:internal/main/run_main_module:23:47","timestamp":"2023-06-17T05:30:58.436Z"}
error: 
    status - 500 
    message - Error: User is already Present. 
    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/services/user.service.js:9:11)
    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)
    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-19T06:10:22.627Z"}
error: 
    status - 500 
    message - Error: User is already Present. 
    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/services/user.service.js:9:11)
    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)
    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-19T06:13:39.872Z"}
error: 
    status - 500 
    message - Error: User is already Present. 
    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/services/user.service.js:9:11)
    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)
    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-19T08:48:24.210Z"}
error: Could not connect to the database. connect ECONNREFUSED 127.0.0.1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:27017\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:847:32)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12","timestamp":"2023-06-21T03:05:17.454Z"}
error: Could not connect to the database. connect ECONNREFUSED 127.0.0.1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:27017\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:847:32)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12","timestamp":"2023-06-21T03:21:48.444Z"}
error: Could not connect to the database. connect ECONNREFUSED 127.0.0.1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:27017\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:847:32)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12","timestamp":"2023-06-21T04:31:33.196Z"}
error: Could not connect to the database. connect ECONNREFUSED 127.0.0.1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:27017\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:847:32)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12","timestamp":"2023-06-21T04:34:16.516Z"}
error: Could not connect to the database. connect ECONNREFUSED 127.0.0.1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:27017\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:847:32)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12","timestamp":"2023-06-21T04:38:43.020Z"}
error: Could not connect to the database. connect ECONNREFUSED 0.0.0.0:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED 0.0.0.0:27017\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:847:32)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12","timestamp":"2023-06-21T04:41:24.314Z"}
error: Could not connect to the database. Invalid message size: 1347703880, max allowed: 67108864 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: Invalid message size: 1347703880, max allowed: 67108864\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:847:32)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12","timestamp":"2023-06-21T06:25:17.690Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-21T06:30:09.540Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-21T06:32:01.327Z"}
error: Could not connect to the database. Invalid message size: 1347703880, max allowed: 67108864 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: Invalid message size: 1347703880, max allowed: 67108864\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:847:32)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12","timestamp":"2023-06-21T06:35:33.173Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-21T06:36:30.026Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/users/login 
      method - POST 
      IP - ::1
     {"timestamp":"2023-06-21T11:24:59.429Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid token
    at Object.verify (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/jsonwebtoken/verify.js:75:17)
    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/middlewares/auth.middleware.js:26:29)
    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)
    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12)
    at userAuth (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/middlewares/auth.middleware.js:15:22)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-21T12:15:12.474Z"}
error: uncaughtException: expected 404 to equal 200
AssertionError: expected 404 to equal 200
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:34:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 15:28:02 GMT+0530 (India Standard Time)","error":{"actual":404,"expected":200,"message":"expected 404 to equal 200","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 404 to equal 200\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:34:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[1.31,0.74,0.61],"uptime":29695.53},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":18295811,"external":20689131,"heapTotal":133451776,"heapUsed":100402808,"rss":190894080},"pid":21725,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 404 to equal 200\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:34:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T09:58:02.870Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":34,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: uncaughtException: expected 404 to equal 200
AssertionError: expected 404 to equal 200
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:34:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 15:31:34 GMT+0530 (India Standard Time)","error":{"actual":404,"expected":200,"message":"expected 404 to equal 200","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 404 to equal 200\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:34:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[0.54,0.63,0.6],"uptime":29906.78},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":18765605,"external":21158457,"heapTotal":121778176,"heapUsed":87216760,"rss":182726656},"pid":21904,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 404 to equal 200\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:34:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T10:01:34.117Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":34,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-06-27T10:09:03.778Z"}
error: uncaughtException: expected 500 to equal 400
AssertionError: expected 500 to equal 400
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 15:39:03 GMT+0530 (India Standard Time)","error":{"actual":500,"expected":400,"message":"expected 500 to equal 400","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[1.65,1,0.75],"uptime":30356.47},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":19340699,"external":21733551,"heapTotal":122159104,"heapUsed":92731832,"rss":179564544},"pid":22137,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T10:09:03.807Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":57,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-06-27T10:10:09.782Z"}
error: uncaughtException: expected 500 to equal 400
AssertionError: expected 500 to equal 400
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 15:40:09 GMT+0530 (India Standard Time)","error":{"actual":500,"expected":400,"message":"expected 500 to equal 400","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[2.18,1.24,0.85],"uptime":30422.48},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":19340918,"external":21733790,"heapTotal":122421248,"heapUsed":92488496,"rss":180711424},"pid":22275,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T10:10:09.816Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":57,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-06-27T10:45:10.271Z"}
error: uncaughtException: expected 500 to equal 400
AssertionError: expected 500 to equal 400
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 16:15:10 GMT+0530 (India Standard Time)","error":{"actual":500,"expected":400,"message":"expected 500 to equal 400","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[1.03,0.48,0.18],"uptime":97.57},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":20099092,"external":22491944,"heapTotal":122445824,"heapUsed":98046040,"rss":180367360},"pid":4254,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T10:45:10.296Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":57,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-06-27T11:04:27.146Z"}
error: uncaughtException: expected 500 to equal 400
AssertionError: expected 500 to equal 400
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 16:34:27 GMT+0530 (India Standard Time)","error":{"actual":500,"expected":400,"message":"expected 500 to equal 400","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[0.73,0.73,0.58],"uptime":1254.45},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":19036747,"external":21429619,"heapTotal":129904640,"heapUsed":99060920,"rss":190627840},"pid":5631,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:57:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T11:04:27.172Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":57,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: Could not connect to the database. Invalid connection string {"name":"MongoParseError","stack":"MongoParseError: Invalid connection string\n    at parseConnectionString (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongodb/lib/core/uri_parser.js:585:21)\n    at connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongodb/lib/operations/connect.js:283:3)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongodb/lib/mongo_client.js:284:5\n    at maybePromise (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongodb/lib/utils.js:692:3)\n    at MongoClient.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongodb/lib/mongo_client.js:280:10)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:836:12\n    at new Promise (<anonymous>)\n    at NativeConnection.Connection.openUri (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/connection.js:832:19)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:351:10\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:1149:10)\n    at Mongoose.connect (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mongoose/lib/index.js:350:20)\n    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:12:20)\n    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)\n    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:19:12)\n    at database (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/config/database.js:4:15)\n    at Object.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1257:14)\n    at Module._compile (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/pirates/lib/index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1311:10)\n    at Object.newLoader [as .js] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/pirates/lib/index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:1115:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:962:12)\n    at Module.require (node:internal/modules/cjs/loader:1139:19)\n    at require (node:internal/modules/helpers:121:18)\n    at Object.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:5:1)\n    at Module._compile (node:internal/modules/cjs/loader:1257:14)\n    at Module._compile (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/pirates/lib/index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1311:10)\n    at Object.newLoader [as .js] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/pirates/lib/index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:1115:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:962:12)\n    at Module.require (node:internal/modules/cjs/loader:1139:19)\n    at require (node:internal/modules/helpers:121:18)\n    at Object.exports.requireOrImport (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mocha/lib/esm-utils.js:42:12)\n    at Object.exports.loadFilesAsync (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mocha/lib/esm-utils.js:55:34)\n    at Mocha.loadFilesAsync (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mocha/lib/mocha.js:473:19)\n    at singleRun (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mocha/lib/cli/run-helpers.js:125:15)\n    at exports.runMocha (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mocha/lib/cli/run-helpers.js:190:10)\n    at Object.exports.handler (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/mocha/lib/cli/run.js:362:11)\n    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/yargs/build/index.cjs:443:71","timestamp":"2023-06-27T11:11:59.390Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-06-27T11:12:04.418Z"}
error: uncaughtException: expected 500 to equal 400
AssertionError: expected 500 to equal 400
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 16:42:04 GMT+0530 (India Standard Time)","error":{"actual":500,"expected":400,"message":"expected 500 to equal 400","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[0.72,0.52,0.51],"uptime":1711.71},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":20947820,"external":23340672,"heapTotal":124440576,"heapUsed":98334208,"rss":184467456},"pid":5868,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T11:12:04.437Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":58,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-06-27T11:13:01.626Z"}
error: uncaughtException: expected 500 to equal 400
AssertionError: expected 500 to equal 400
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 16:43:01 GMT+0530 (India Standard Time)","error":{"actual":500,"expected":400,"message":"expected 500 to equal 400","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[0.41,0.46,0.49],"uptime":1768.91},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":20124274,"external":22517126,"heapTotal":124727296,"heapUsed":96358128,"rss":183738368},"pid":6005,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T11:13:01.636Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":58,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-06-27T11:46:20.067Z"}
error: uncaughtException: expected 500 to equal 400
AssertionError: expected 500 to equal 400
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 17:16:20 GMT+0530 (India Standard Time)","error":{"actual":500,"expected":400,"message":"expected 500 to equal 400","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[1.76,1.22,1.21],"uptime":3767.36},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":19207438,"external":21600758,"heapTotal":125288448,"heapUsed":97634888,"rss":188338176},"pid":7229,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T11:46:20.082Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":58,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-06-27T11:47:39.977Z"}
error: uncaughtException: expected 500 to equal 400
AssertionError: expected 500 to equal 400
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 17:17:39 GMT+0530 (India Standard Time)","error":{"actual":500,"expected":400,"message":"expected 500 to equal 400","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[1.68,1.32,1.24],"uptime":3847.26},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":19279874,"external":21672726,"heapTotal":123191296,"heapUsed":92702240,"rss":184094720},"pid":7352,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T11:47:39.987Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":58,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-06-27T11:48:00.303Z"}
error: uncaughtException: expected 500 to equal 400
AssertionError: expected 500 to equal 400
    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)
    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)
    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)
    at Object.onceWrapper (node:events:625:28)
    at Server.emit (node:events:511:28)
    at emitCloseNT (node:net:2223:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Tue Jun 27 2023 17:18:00 GMT+0530 (India Standard Time)","error":{"actual":500,"expected":400,"message":"expected 500 to equal 400","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[1.42,1.28,1.23],"uptime":3867.59},"process":{"argv":["/usr/bin/node","/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/.bin/mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/integration/user.test.js","tests/unit/user.test.js"],"cwd":"/home/akhil/CFP238/Fundoo_Note/FundooNote","execPath":"/usr/bin/node","gid":1000,"memoryUsage":{"arrayBuffers":19219564,"external":21612884,"heapTotal":124805120,"heapUsed":94064168,"rss":181157888},"pid":7437,"uid":1000,"version":"v20.3.1"},"stack":"AssertionError: expected 500 to equal 400\n    at Test.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js:58:40)\n    at Test.assert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:181:6)\n    at Server.localAssert (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js:131:12)\n    at Object.onceWrapper (node:events:625:28)\n    at Server.emit (node:events:511:28)\n    at emitCloseNT (node:net:2223:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-06-27T11:48:00.314Z","trace":[{"column":40,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/tests/integration/user.test.js","function":null,"line":58,"method":null,"native":false},{"column":6,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/supertest/lib/test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":625,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":511,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":2223,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/validators/user.validator.js:12:16)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at Function.handle (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:175:3)
    at router (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at logger (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:328:13)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:346:12)
    at next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/express/lib/router/index.js:280:10)
    at /home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/body-parser/lib/read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:231:16)
    at done (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:511:28)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1367:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-27T11:50:17.801Z"}
error: 
    status - 500 
    message - Error: User is already Present. 
    at _callee$ (/home/akhil/CFP238/Fundoo_Note/FundooNote/src/services/user.service.js:12:11)
    at tryCatch (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:125:22)
    at Generator.next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/home/akhil/CFP238/Fundoo_Note/FundooNote/node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-28T03:47:10.820Z"}
